src/lib/GovernorTimelockCompound.sol

```diff
@@ -0,0 +1,246 @@
+// SPDX-License-Identifier: MIT
+// OpenZeppelin Contracts (last updated v4.6.0) (governance/extensions/GovernorTimelockCompound.sol)
+//
+// The Radworks Timelock is not compatible with the Governor Alpha Timelock interface. This
```

Comment:
Nit: took a shot at rewording this, feel free to ignore

> This `GovernorTimelockCompound` has been modified to support a Radworks Timelock which does not conform to Governor Alpha Timelock. If Radworks decides to change the Timelock in the future...

Also, wondering if there's sense in renaming this file and contract name to better express that it was modified.

Reply:
Done.. and agree on the name change.. also done 4c8b4aa5079ff2ca8a0124eeab3624cc5cfce0db